src/BUILD.gn

```diff
@@ -45,7 +45,7 @@ config("includes") {
   defines = [ "CHIP_HAVE_CONFIG_H=1" ]
 
   # Make __FILE__ and related macros relative to chip_root
-  cflags = [ "-fmacro-prefix-map=${build_relative_chip_root}/=" ]
+  # cflags = [ "-fmacro-prefix-map=${build_relative_chip_root}/=" ]
```

Comment:
This will expose full paths in error messages which is (1) a privacy leak and (2) larger code.